compose/docker-compose.base.yml

```
@@ -11,8 +11,30 @@ networks:
+    driver: local
+    name: pg-admin
+
+  drone-ci:
+    driver: local
+    name: drone-ci
+
+  metabase-data:
+    driver: local
+    name: metabase
+
+  redis-data: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)